.build/nais-prod.yaml

```
@@ -39,6 +39,8 @@ spec:
         - application: familie-ef-sak-frontend
         - application: familie-ba-sak
           cluster: prod-gcp
+        - application: familie-ks-sak
+          cluster: prod-gcp
```

Då denne allerede kjører i prod-gcp så trenger man ikke å sette prod-gcp i cluster :)
cluster settes til det samme som applikasjonen sitt hvis det ikke er definiert
```suggestion
        - application: familie-ks-sak
```